src/components/nav/NavBar.js

```diff
@@ -1,27 +1,55 @@
-import { useRef } from "react"
-import { Link, useNavigate } from "react-router-dom"
-import "./NavBar.css"
-import Logo from "./rare.jpeg"
+import { useRef } from "react";
+import { Link, useNavigate } from "react-router-dom";
+import "./NavBar.css";
+import Logo from "./rare.jpeg";
 
+// Creates and exports NavBar component
+// uses Props in the argument defined in Rare.js
 export const NavBar = ({ token, setToken }) => {
-  const navigate = useNavigate()
-  const navbar = useRef()
-  const hamburger = useRef()
+  //defines navigate variable to use useNavigate hook
+  const navigate = useNavigate();
 
+  /*
+  •useRefs() hook are used for functionality of the mobile nav bar, which appears when viewport width is decreased
```

Comment:
Please clarify this comment about useRefs()

Reply:
done 👍